client/src/extension.ts

```diff
@@ -12,6 +12,7 @@ import * as vscode from 'vscode';
 import * as lsp from 'vscode-languageclient';
 
 import {registerCommands} from './commands';
+import {ProjectLoadingNotificationService} from './project_loading_notification_service';
 import {projectLoadingNotification} from './protocol';
```

Comment:
Can we put `projectLoadingNotification` and `ProjectLoadingNotificationService` in the same source file?